src/nav/sim/render.py

```
@@ -517,24 +606,140 @@ def _render_combined_model_cached(
+
+    ref_center_v = size_v / 2.0
+    ref_center_u = size_u / 2.0
+
+    # Build order_near_to_far for bodies (needed for body_index_map)
+    sorted_bodies_by_range = sorted(bodies_with_ranges, key=lambda x: x['range'])
+    order_near_to_far = [
+        bp.get('name', f'SIM-BODY-{i+1}').upper()
+        for i, bp in enumerate(sorted_bodies_by_range)
+    ]
+
+    for range_val, item_type, item_params, orig_idx in render_items:
```

Ruff (0.14.11)

Loop control variable `range_val` not used within loop body

Rename unused `range_val` to `_range_val`

(B007)